src/palindrom-network-channel.js

```diff
@@ -44,7 +45,6 @@ export default class PalindromNetworkChannel {
         onReceive && (this.onReceive = onReceive);
         onSend && (this.onSend = onSend);
         onConnectionError && (this.onConnectionError = onConnectionError);
-        onFatalError && (this.onFatalError = onFatalError);
```

Comment:
Originally `onConnectionError` was used to display a message to the user saying that there are temporary connection problems but we are reconnecting. Similar to:

- ![image](https://user-images.githubusercontent.com/566463/60732131-8daf0f80-9f49-11e9-91c2-2a4154775669.png)
 
On the other hand, `onFatalError` was used to display Palindrom crashes that can only be solved by reloading the page. Similar to:

- ![image](https://user-images.githubusercontent.com/566463/60732190-ae776500-9f49-11e9-8fe9-efd46f761b3e.png)

After this change, these two cases are undistinguished.

Reply:
Then it seems those cases were broken before, unless I miss something.

I didn;t change the functionality, I just moved the error handling from Palindrom https://github.com/Palindrom/Palindrom/pull/252/files#diff-92b698a38d3db0381546f440a80e631bL298 
to NetworkChannel https://github.com/Palindrom/Palindrom/pull/252/files#diff-09a771a96432511bf7bc17a34547183dR119

I cannot find any other use of `palindromInstance.onFatalError` in any Palindrom custom-element, https://github.com/search?p=1&q=org%3APalindrom+onFatalError&type=Code
Also, there is no specs nor docs for `onFatalError` usage, so I considered it not public API https://palindrom.github.io/docs/search.html?q=onFatalError

Reply:
I agree, let's cleanup the existence of `onFatalError` and re-add it in future in needed.